pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.pytest.ini_options]
+addopts = "--ignore=continuous_integration --ignore=docs --ignore=.github"
```

Comment:
We're not testing code snippets in the docs today, but wanted to flag that `--ignore=docs` _may_ need to be removed if we want to start that level of testing